### ISSUE TITLE
BC220CF: screw holes in IEC_Housing.stl are too large for M3

### ISSUE THREAD

**Issue by daduke** [martinbudden/BabyCube#9]

**Comment by martinbudden** [comment]
Yes, they are big enough to fit threaded inserts. I don't normally like to use threaded inserts, but I figured that for the IEC housing they shoulld be used, as a safety thing.

I guess the build instructions need to be updated to reflect this.
